src/protocol.h

```diff
@@ -235,8 +248,6 @@ extern const char* GETCFCHECKPT;
 /**
  * cfcheckpt is a response to a getcfcheckpt request containing a vector of
  * evenly spaced filter headers for blocks on the requested chain.
- * Only available with service bit NODE_COMPACT_FILTERS as described by
- * BIP 157 & 158.
```

Comment:
Why are you removing this?

Reply:
It doesn't appear in the comments for `cfheaders` and `cfilter`, so I'm removing it from here for consistency. It seems obvious that `NODE_COMPACT_FILTERS` is required since it's a response to `getcfcheckpt`, which requires `NODE_COMPACT_FILTERS`.

Reply:
Fair enough